mlir/test/Dialect/Vector/canonicalize.mlir

```diff
@@ -60,7 +60,7 @@ func.func @create_vector_mask_to_constant_mask_scalable_all_true() -> (vector<8x
   %c16 = arith.constant 16 : index
   %0 = vector.vscale
   %1 = arith.muli %0, %c16 : index
-  // CHECK: vector.constant_mask [8, 16] : vector<8x[16]xi1>
```

Comment:
Errr... this original output doesn't look correct to me... @banach-space? Do we have a bug somewhere else?